csrc/python_frontend/python_bindings.cpp

```diff
@@ -421,6 +421,27 @@ void initNvFuserPythonBindings(PyObject* module) {
       .value("unswitch", ParallelType::Unswitch)
       .value("vectorize", ParallelType::Vectorize);
 
+  //! LoadStoreOpType used for scheduling
+  py::enum_<LoadStoreOpType>(nvfuser, "LoadStoreOpType")
+      .value("set", LoadStoreOpType::Set)
+      .value("load_matrix", LoadStoreOpType::LdMatrix)
+      .value("load_matrix_transpose", LoadStoreOpType::LdMatrixTranspose)
+      .value("cp_async", LoadStoreOpType::CpAsync)
+      .value("tma", LoadStoreOpType::CpAsyncBulkTensorTile);
+
+  //! CacheOp used for scheduling
+  py::enum_<CacheOp>(nvfuser, "CacheOp")
+      .value("none", CacheOp::Unspecified)
```

Comment:
nitpick, should we stick with `unspecified` instead?

Reply:
🤷🏼‍♂️ Lol. I can't consistently spell `unspecified` without an IDE.